configtx/orderer.go

```diff
@@ -62,6 +61,18 @@ type OrdererOrg struct {
 	name     string
 }
 
+// OrdererOrgMSP encapsulates the configuration functions used to modify an orderer organization's MSP.
+type OrdererOrgMSP struct {
```

Comment:
This type is unused now and can be deleted.

Reply:
🛠️